hwc_names.hpp

```diff
@@ -2978,271 +2978,272 @@ static const char *const hardware_counters_mali_tSIx[] =
-		"TNOx_L2_WR_MSG_IN",
-		"TNOx_L2_WR_MSG_IN_STALL",
-		"TNOx_L2_SNP_MSG_IN",
-		"TNOx_L2_SNP_MSG_IN_STALL",
-		"TNOx_L2_RD_MSG_OUT",
-		"TNOx_L2_RD_MSG_OUT_STALL",
-		"TNOx_L2_WR_MSG_OUT",
-		"TNOx_L2_ANY_LOOKUP",
-		"TNOx_L2_READ_LOOKUP",
-		"TNOx_L2_WRITE_LOOKUP",
-		"TNOx_L2_EXT_SNOOP_LOOKUP",
-		"TNOx_L2_EXT_READ",
-		"TNOx_L2_EXT_READ_NOSNP",
-		"TNOx_L2_EXT_READ_UNIQUE",
-		"TNOx_L2_EXT_READ_BEATS",
-		"TNOx_L2_EXT_AR_STALL",
-		"TNOx_L2_EXT_AR_CNT_Q1",
-		"TNOx_L2_EXT_AR_CNT_Q2",
-		"TNOx_L2_EXT_AR_CNT_Q3",
-		"TNOx_L2_EXT_RRESP_0_127",
-		"TNOx_L2_EXT_RRESP_128_191",
-		"TNOx_L2_EXT_RRESP_192_255",
-		"TNOx_L2_EXT_RRESP_256_319",
-		"TNOx_L2_EXT_RRESP_320_383",
-		"TNOx_L2_EXT_WRITE",
-		"TNOx_L2_EXT_WRITE_NOSNP_FULL",
-		"TNOx_L2_EXT_WRITE_NOSNP_PTL",
-		"TNOx_L2_EXT_WRITE_SNP_FULL",
-		"TNOx_L2_EXT_WRITE_SNP_PTL",
-		"TNOx_L2_EXT_WRITE_BEATS",
-		"TNOx_L2_EXT_W_STALL",
-		"TNOx_L2_EXT_AW_CNT_Q1",
-		"TNOx_L2_EXT_AW_CNT_Q2",
-		"TNOx_L2_EXT_AW_CNT_Q3",
-		"TNOx_L2_EXT_SNOOP",
-		"TNOx_L2_EXT_SNOOP_STALL",
-		"TNOx_L2_EXT_SNOOP_RESP_CLEAN",
-		"TNOx_L2_EXT_SNOOP_RESP_DATA",
-		"TNOx_L2_EXT_SNOOP_INTERNAL",
-		"",
-		"",
-		"",
-		"",
-		"",
-		"",
-		"",
-		};
+        /* Performance counters for the Memory System */
+        "",
+        "",
+        "",
+        "",
+        "TNOx_MMU_REQUESTS",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "TNOx_L2_RD_MSG_IN",
+        "TNOx_L2_RD_MSG_IN_STALL",
+        "TNOx_L2_WR_MSG_IN",
+        "TNOx_L2_WR_MSG_IN_STALL",
+        "TNOx_L2_SNP_MSG_IN",
+        "TNOx_L2_SNP_MSG_IN_STALL",
+        "TNOx_L2_RD_MSG_OUT",
+        "TNOx_L2_RD_MSG_OUT_STALL",
+        "TNOx_L2_WR_MSG_OUT",
+        "TNOx_L2_ANY_LOOKUP",
+        "TNOx_L2_READ_LOOKUP",
+        "TNOx_L2_WRITE_LOOKUP",
+        "TNOx_L2_EXT_SNOOP_LOOKUP",
+        "TNOx_L2_EXT_READ",
+        "TNOx_L2_EXT_READ_NOSNP",
+        "TNOx_L2_EXT_READ_UNIQUE",
+        "TNOx_L2_EXT_READ_BEATS",
+        "TNOx_L2_EXT_AR_STALL",
+        "TNOx_L2_EXT_AR_CNT_Q1",
+        "TNOx_L2_EXT_AR_CNT_Q2",
+        "TNOx_L2_EXT_AR_CNT_Q3",
+        "TNOx_L2_EXT_RRESP_0_127",
+        "TNOx_L2_EXT_RRESP_128_191",
+        "TNOx_L2_EXT_RRESP_192_255",
+        "TNOx_L2_EXT_RRESP_256_319",
+        "TNOx_L2_EXT_RRESP_320_383",
+        "TNOx_L2_EXT_WRITE",
+        "TNOx_L2_EXT_WRITE_NOSNP_FULL",
+        "TNOx_L2_EXT_WRITE_NOSNP_PTL",
+        "TNOx_L2_EXT_WRITE_SNP_FULL",
+        "TNOx_L2_EXT_WRITE_SNP_PTL",
+        "TNOx_L2_EXT_WRITE_BEATS",
+        "TNOx_L2_EXT_W_STALL",
+        "TNOx_L2_EXT_AW_CNT_Q1",
+        "TNOx_L2_EXT_AW_CNT_Q2",
+        "TNOx_L2_EXT_AW_CNT_Q3",
+        "TNOx_L2_EXT_SNOOP",
+        "TNOx_L2_EXT_SNOOP_STALL",
+        "TNOx_L2_EXT_SNOOP_RESP_CLEAN",
+        "TNOx_L2_EXT_SNOOP_RESP_DATA",
+        "TNOx_L2_EXT_SNOOP_INTERNAL",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+};
```

Comment:
Indentation seems correct now, for consistency we only need to move the closing brace up to the last line

Reply:
Oh sorry I see it has already been merged, you can ignore this then